miniapps/plasma/cold_plasma_dielectric_solver.cpp

```diff
@@ -1151,31 +1196,47 @@ CPDSolver::Solve()
       {
          rhs.imag() *= -1.0;
       }
+      rhs.real().SyncAliasMemory(rhs);
+      rhs.imag().SyncAliasMemory(rhs);
+      tmp.real().SyncAliasMemory(tmp);
+      tmp.imag().SyncAliasMemory(tmp);
```

Comment:
Let's add a method to `ParComplexLinearForm` to combine these lines a little

Reply:
I see this could be of interest to both [Par]ComplexLinearForm and [Par]ComplexGridFunction. Should I add this in https://github.com/mfem/mfem/pull/1575 instead (similar calls are made there in `complex_fem.cpp`)?

Reply:
Yes, that looks like the best place

Reply:
Okay, I will make the changes in #1575 and then pull it into this branch so that the code above can use it.

Reply:
See https://github.com/mfem/mfem/pull/1575/commits/4be4d220a8db245018a272d517cf26e94cef854c.